.github/workflows/build.yml

```diff
@@ -28,8 +28,10 @@ jobs:
             shorthand: etwh
           - name: heatpump_water_heater
             shorthand: hpwh
-          - name: hvac
-            shorthand: hvac
+          - name: hvac_furnace
+            shorthand: hvac_furnace
+          - name: hvac_air_handler
+            shorthand: hvac_air_handler
```

Comment:
This is going to break the webtools, which will expect an `hvac` device to exist. So we'll need to update that to either pick one of these by default or expose "HVAC with Furnace" and "HVAC with Air Handler" as different options.

See https://github.com/esphome-econet/esphome-econet.github.io/blob/main/_includes/improvSerialButton.html

Reply:
See https://github.com/esphome-econet/esphome-econet.github.io/issues/5